_security-analytics/sec-analytics-config/detectors-config.md

```diff
@@ -40,28 +41,28 @@ You can define a new detector by naming the detector, selecting a data source an
     To quickly select one or more known rules and dismiss others, first deselect all rules by moving the **Rule name** toggle to the left, then search for your target rule names and select each individually by moving its toggle to the right.
     {: .tip }
 
-1. In the **Detector schedule** section, set how often the detector will run. Specify a unit of time and a corresponding number to set the interval.
-1. Choose **Next** in the lower-right corner of the screen to continue. The **Configure field mapping** page appears.
 
 ## Step 2. Create field mappings
 
 The field mapping step matches field names from the rule with field names from the log index being used to provide data. Creating field mappings allows the system to accurately pass event data from the log to the detector and then use the data to trigger alerts.
 
-The data source (log index), log type, and detection rules specified in the first step determine which fields are available for mapping. For example, when "Windows logs" is selected as the log type, this parameter, along with the specific detection rules, determines the list of rule field names available for the mapping. Similarly, the selected data source (log index) determines the list of log source field names that are available for the mapping.
+The data source (log index), log type, and detection rules specified in the first step determine which fields are available for mapping. For example, when "Windows logs" is selected as the log type, this parameter, along with the specific detection rules, determines the list of detection field names available for the mapping. Similarly, the selected data source determines the list of log source field names that are available for the mapping.
+
+The system uses prepackaged Sigma rules for detector creation and it can automatically map important fields for a specific log type with the corresponding fields in the Sigma rules. The field mapping step presents a view of automatically mapped fields while also providing the option to customize, change, or add new field mappings. When a detector includes customized rules, you can follow this step to manually map rule field names to log source field names.
```

Comment:
```suggestion
The system uses prepackaged Sigma rules for detector creation and it can automatically map important fields for a specific log type with the corresponding fields in the Sigma rules. The field mapping step presents a view of automatically mapped fields while also providing the option to customize, change, or add new field mappings. When a detector includes customized rules, you can follow this step to manually map the field names of rules to field names of log sources.
```

Reply:
This section gets marked for changes by different reviewers every version release, with different reviewers making suggested changes to content that was changed based on suggestions in previous versions. It is perpetually being suggested for change, and I suspect it will continue to get reviewed and blessed with fresh suggested changes each and every subsequent version of OpenSearch until the end of time. In fact, this phenomenon is somewhat inherent in the doc review process.

Anyhow, in this case, I think it's better to stay close to the syntax used in the UI labels, i.e., "Detector field name", 'log source field name".

Reply:
In this case, I think it's better to stay close to the syntax used in the UI labels, i.e., "Detector field name", 'log source field name". Although I did change "rule" to "detector", even if they are synonymous in this application.